.npmignore

```diff
@@ -1,3 +0,0 @@
-test/
```

Comment:
I don't know if there's any value in pushing the extra bytes of test files to everyone via npm

Reply:
I removed the `.npmignore` file in favor of using the `"files"` field in the `package.json` file.

The `"files"` field works as a whitelist as apposed to a blacklist like the `.npmignore` file.

I've got the files set to the following (2 folders and 3 files (plus the package.json file itself)):

```json
"files": [
  "dist",
  "types",
  "changelog.md",
  "license.txt",
  "readme.md"
],
```

You can run `npm pack` to create the package locally so you can see what's included in it.

Reply:
Ah cool, yeah, I like `files` for that